### ISSUE TITLE
Slide on folders

### ISSUE THREAD

**Issue by ServerReset** [Aliucord/plugin-requests#1101]
### Plugin Description

I want to be able to slide vertically on a folder icon to unfold the folder and quickly switch to one of the servers inside of it, when you let go it would switch to it and collapse the folder. Like how if you hold the three dots in Chrome and slide it brings up the drop-down menu and when you let go it selects that item

https://github.com/Aliucord/plugin-requests/assets/30058771/8d982f5c-85d3-425e-8780-c456dc4f5a85

### Request Agreement

- [X] I have made sure my plugin is possible and abides by the rules!

### ​

- [X] I did indeed check to make sure my plugin request is original!

**Comment by rushiiMachine** [comment]
is this even doable in android

**Comment by admin-elvistrujanovic** [comment]
> is this even doable in android

Yes it is - if you want to add core features from Discord React Native updated client to this kotlin client

**Comment by Ushie** [comment]
Nope
